geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -317,6 +349,18 @@ pub trait GeyserPlugin: Any + Send + Sync + std::fmt::Debug {
         Ok(())
     }
 
+    /// Called when a cluster info is updated on gossip network.
+    #[allow(unused_variables)]
+    fn update_cluster_info(&self, cluster_info: &ReplicaClusterInfoNode) -> Result<()> {
+        Ok(())
+    }
+
+    /// Called when a cluster info is removed on gossip network.
+    #[allow(unused_variables)]
+    fn notify_clusterinfo_remove(&self, pubkey: &Pubkey) -> Result<()> {
```

Comment:
When is this called? notify_clusterinfo_remove?

Reply:
You can find the call here: https://github.com/solana-labs/solana/blob/f4bc1be708298c0896978c1ce286dcdbf1ecc221/gossip/src/crds.rs#L554